test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -ex
+
+pip install tox
```

Comment:
Do we need this?  Looks like the container you are using already installs tox at build time:  https://github.com/higebu/docker-tox/blob/master/install.sh#L37